study-builder/studies/pancan/i18n/en.conf

```diff
@@ -17,21 +17,179 @@
 "prequal_sections_01_question_01_picklistOptions_02_optionLabelTemplate"="My child has been diagnosed with cancer and signing up with them or for them"
 "prequal_sections_01_question_01_validations_hintTemplate"="Please choose one of the above options."
 
-"prequal_diagnosed_primary_cancers_prompt"="What primary cancer(s) have you/your child been diagnosed with?"
-"prequal_sections_02_question_01_validations_01_hintTemplate_01"="If you/your child have been diagnosed with more than one cancer type, please select all that apply."
+"prequal_diagnosed_primary_cancer_list_prompt"="What primary cancer(s) have you/your child been diagnosed with? \n If you/your child have been diagnosed with more than one cancer type, please use the button below to add an additional diagnosis."
+"prequal_sections_02_question_01_validations_01_hintTemplate_01"="If you/your child have been diagnosed with more than one cancer type, please use the button below to add an additional diagnosis."
```

Comment:
Can you cleanup these variable names and use JSON-like syntax so it's more consistent?